.github/workflows/assign-buddy.yml

```
@@ -0,0 +1,39 @@
+on:
+    pull_request:
+        types: [opened, ready_for_review]
+
+jobs:
+    assign-reviewer:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v4
+            - name: Assign Buddy
+              uses: actions/github-script@v6
```

actionlint (1.7.11)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)